doc/source/diagnostics/process_list.rst

```diff
@@ -41,7 +41,7 @@ INFORMATION_SCHEMA Tables
    :column STATE: An action, event, or state that indicates what the thread is doing.
    :column INFO: The statement that the thread is executing, or NULL if it is not executing any statement.
    :column TIME_MS: The time in milliseconds that the thread has been in its current state.
-   :column ROWS_EXAMINED: The number of rows examined by the statement being executed.
+   :column ROWS_EXAMINED: The number of rows examined by the statement being executed (*NOTE:* This column is not updated for each examined row so it does not necessarily show an up-to-date value while the statement is executing.).
```

Comment:
... while the statement is executing. It only shows a correct value after the statement has completed.